core/src/main/java/org/apache/carbondata/core/scan/result/vector/impl/directread/ColumnarVectorWrapperDirectWithDeleteDelta.java

```diff
@@ -41,6 +41,31 @@ public ColumnarVectorWrapperDirectWithDeleteDelta(CarbonColumnVector vectorWrapp
     this.nullBits = nullBits;
   }
 
+  @Override
+  public CarbonColumnVector getColumnVector() {
+    return null;
```

Comment:
Throw unsupported exception for all these

Reply:
And all the below files where it is not applicable

Reply:
Also check if it is required by the majority, if not. just modify in implementation not the interface.